config/filetypes.yml

```diff
@@ -408,15 +428,18 @@ unimportant:
     - .CFUserTextEncoding
     - .DS_Store
     - .localized
+    - Icon\r
```

Comment:
I think this is a mistake?

Reply:
Oh, you said it's needed in the description. Why?

Reply:
I don't know exactly why the filename contains a carriage return, but some more information can be found here: https://superuser.com/a/298798.

Reply:
And can you confirm that this works properly with `vivid` + GNU ls?

Reply:
~/.config/vivid/filetypes.yml:
```yaml
test:
  - Icon\r
```
~/.config/vivid/one-dark-simple.yml:
```yaml
...
test:
  foreground: red
```

![image](https://github.com/sharkdp/vivid/assets/23235841/b2360fbf-010e-4308-bdbb-817dab41dbd2)

Reply:
Tested on MacOS, gls v9.3, fish

Reply:
Thank you